src/PrestaShopBundle/Resources/config/routing/admin/sell/catalog/attribute.yml

```diff
@@ -26,16 +26,6 @@ admin_attributes_search:
     redirectQueryParamsToKeep:
       - 'attributeGroupId'
 
-admin_attributes_create:
```

Comment:
I'm not sure this should be moved, the route is about Attribute, it's bound to AttributeController so it better fits in this file The fact that the legacy controller is `AdminAttributesGroups` is misleading but it makes sense because in the legacy page there was a unique controller to handle both values and groups

So they share the same parent legacy controller and the same permission but we can keep the routes apart